release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -695,6 +695,12 @@ A new VRF CNI plugin that allows you to assign additional networks to a VRF is n
 
 For more information, see xref:../networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc#cnf-assigning-a-secondary-network-to-a-vrf[Assigning a secondary network to a VRF] and xref:../networking/hardware_networks/configuring-sriov-device.adoc#cnf-assigning-a-sriov-network-to-a-vrf_configuring-sriov-device[Assigning an SR-IOV network to a VRF].
 
+[id="ocp-4-7-scale-xt_32-end-to-end-test"]
+==== The xt_u32 end-to-end test is enabled for CNF
+xt_u32 is an iptables kernel module that allows packet filtering based on arbitrary content. It can look beyond headers or special protocols that are not covered by other iptables modules. Some of the Telco 5G partners make extensive use of this module in their software to do rule based packet routing.
+
+For more information, see xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-for-platform-verification_cnf-master[Performing end-to-end tests for platform verification].
```

Comment:
Replace the .html in the file path with .adoc 

I would also drop the sentence about 5G partners; we don't typically discuss partner behavior in our doc set.